app/Http/Controllers/SettingsController.php

```diff
@@ -56,6 +55,7 @@ class SettingsController
         'emotions_primary',
         'emotions_secondary',
         'failed_jobs',
+        'gift_photo',
```

Comment:
If we merge the foreign keys PR, we don't need this anymore. However there will be some work to do in the Export account script, I think.